src/primitives.ts

```diff
@@ -28,9 +28,9 @@ export enum EventName {
   NFT_FILTER_SELECTED = 'NFT Filter Selected',
   NFT_LISTING_COMPLETED = 'NFT Listing Success',
   NFT_LISTING_SIGNED = 'NFT Listing Signed',
+  NFT_PROFILE_PAGE_START_SELL = 'NFT Profile Page Start Sell',
   NFT_SELL_ITEM_ADDED = 'NFT Sell Item Added',
-  NFT_SELL_SELECTED = 'NFT Sell Selected',
```

Comment:
this event is no longer needed because the sell button toggle on view my nfts page doesnt exist in the UI design anymore